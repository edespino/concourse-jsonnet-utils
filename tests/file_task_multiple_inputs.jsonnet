local concourse = import 'concourse.libsonnet';

concourse.FileTask('pipeline/tasks/build/task.sh', inputs = ['git', 'dev-tools'])
