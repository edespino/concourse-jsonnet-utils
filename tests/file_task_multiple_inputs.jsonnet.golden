{
   "inputs": [
      "git",
      "dev-tools"
   ],
   "platform": "linux",
   "run": {
      "dir": "git",
      "path": "pipeline/tasks/build/task.sh"
   }
}
